src/main/scala/mesosphere/marathon/upgrade/TaskReplaceActor.scala

```diff
@@ -138,7 +138,7 @@ object TaskReplaceActor {
   private[this] val log = LoggerFactory.getLogger(getClass)
 
   val KillComplete = "^TASK_(ERROR|FAILED|FINISHED|LOST|KILLED)$".r
```

Comment:
Since `KillComplete` and `FailedToStart` are now equal, I would use one and call it smth. like `IsTerminal`.